lib/glific/flows/action.ex

```diff
@@ -732,9 +732,11 @@ defmodule Glific.Flows.Action do
   end
 
   def execute(%{type: "set_run_result"} = action, context, messages) do
+    value = ContactField.parse_contact_field_value(context, action.value)
```

Comment:
Let's use FlowContext.parse_value function. We should avoid other funcations or may be convert them
As a private funcation.

Reply:
made requested changes